tutorials/scripting/c_sharp/c_sharp_features.rst

```diff
@@ -144,8 +144,8 @@ It is possible to bind values when establishing a connection by passing an objec
         var plusButton = (Button)GetNode("PlusButton");
         var minusButton = (Button)GetNode("MinusButton");
 
-        plusButton.Connect("pressed", this, "ModifyValue", new object[] { 1 });
-        minusButton.Connect("pressed", this, "ModifyValue", new object[] { -1 });
+        plusButton.Connect("pressed", this, "ModifyValue", new Godot.Collections.Array(1));
+        minusButton.Connect("pressed", this, "ModifyValue", new Godot.Collections.Array(-1));
```

Comment:
The correct syntax is:

```suggestion
        plusButton.Connect("pressed", this, "ModifyValue", new Godot.Collections.Array { 1 });
        minusButton.Connect("pressed", this, "ModifyValue", new Godot.Collections.Array { -1 });
```

Reply:
Doesn't either one work? This is using the `Array(params object[] array)` constructor.

Reply:
I don't know why that constructor was added (I approved the PR but somehow missed that detail). I'm asking on our chat to find its purpose, but I prefer the braces initialization syntax overall.